.github/workflows/build.yml

```diff
@@ -433,3 +433,41 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           tag_name: ${{ steps.extract_tag.outputs.tag_name }}
           files: ./projects/*.zip
+
+  macos:
+    runs-on: macos-latest
```

Comment:
Building for latest macos can make older versions not compatible. Since latest macos is only arm. The only intel build available is 10.13.